bricks/permission_handler/__brick__/{{~ _Podfile_build_configurations }}

```diff
@@ -0,0 +1,52 @@
+      # TODO You can enable the permissions needed here.
+      # permission, just remove the `#` character in front so it looks like this:
+      #
+      # ## dart: PermissionGroup.camera
+      #'PERMISSION_CAMERA=1'
+      #
+      #  Preprocessor definitions can be found in: https://github.com/Baseflow/flutter-permission-handler/blob/master/permission_handler/ios/Classes/PermissionHandlerEnums.h
+      config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= [
+      '$(inherited)',
+
+      ## dart: PermissionGroup.calendar
+      # 'PERMISSION_EVENTS=1',
+
+      ## dart: PermissionGroup.reminders
+      # 'PERMISSION_REMINDERS=1',
+
+      ## dart: PermissionGroup.contacts
+      # 'PERMISSION_CONTACTS=1',
+
+      ## dart: PermissionGroup.camera
+      # 'PERMISSION_CAMERA=1',
+
+      ## dart: PermissionGroup.microphone
+      # 'PERMISSION_MICROPHONE=1',
+
+      ## dart: PermissionGroup.speech
+      # 'PERMISSION_SPEECH_RECOGNIZER=1',
+
+      ## dart: PermissionGroup.photos
+      # 'PERMISSION_PHOTOS=1',
+
+      ## dart: [PermissionGroup.location, PermissionGroup.locationAlways, PermissionGroup.locationWhenInUse]
+      # 'PERMISSION_LOCATION=1',
+
+      ## dart: PermissionGroup.notification
+      # 'PERMISSION_NOTIFICATIONS=1',
+
+      ## dart: PermissionGroup.mediaLibrary
+      # 'PERMISSION_MEDIA_LIBRARY=1',
+
+      ## dart: PermissionGroup.sensors
+      # 'PERMISSION_SENSORS=1',
+
+      ## dart: PermissionGroup.bluetooth
+      # 'PERMISSION_BLUETOOTH=1',
+
+      ## dart: PermissionGroup.appTrackingTransparency
+      # 'PERMISSION_APP_TRACKING_TRANSPARENCY=1',
+
+      ## dart: PermissionGroup.criticalAlerts
+      # 'PERMISSION_CRITICAL_ALERTS=1'
+      ]
```

Comment:
@manh-t Should we add more line end of file here? 🤔

Reply:
@doannimble Nope, as we need to keep exactly that content to be injected into the original file `bricks/template/__brick__/{{project_name.snakeCase()}}/ios/Podfile`